### ISSUE TITLE
运行出错，所有模型都安装了

### ISSUE THREAD

**Issue by wuzhonghui56** [bytedance/ComfyUI-HyperLoRA#16]
![Image](https://github.com/user-attachments/assets/56741e87-0250-4d02-8236-9d5f6c301e58)

![Image](https://github.com/user-attachments/assets/3f807b06-4bca-49ba-824b-f2daae2e3251)

**Comment by wuzhonghui56** [comment]
很长一串错误提示，省略了中间都是那些尺寸的错误
got prompt
model weight dtype torch.float16, manual cast: None
model_type EPS
Using xformers attention in VAE
Using xformers attention in VAE
VAE load device: cuda:0, offload device: cpu, dtype: torch.bfloat16
Requested to load SDXLClipModel
loaded completely 9.5367431640625e+25 1560.802734375 True
CLIP/text encoder model load device: cuda:0, offload device: cpu, current: cuda:0, dtype: torch.float16
Requested to load SDXLClipModel
Value of key "pretrained_model" was changed from "None" to "E:\ComfyUI_flux_redux\ComfyUI\models\hyper_lora/clip_processor\clip_vit_large_14_processor"
Value of key "pretrained_model" was changed from "None" to "E:\ComfyUI_flux_redux\ComfyUI\models\hyper_lora/clip_vit\clip_vit_large_14"
Value of key "encoder_types" was changed from "['clip']" to "['clip', 'arcface']"
Value of key "insightface_root" was changed from "./" to "E:\ComfyUI_flux_redux\ComfyUI\models\insightface"
Value of key "num_id_tokens" was changed from "4" to "16"
Value of key "hyper_dim" was changed from "1024" to "128"
Value of key "lora_rank" was changed from "4" to "8"
image_processor:
  pretrained_model: E:\ComfyUI_flux_redux\ComfyUI\models\hyper_lora/clip_processor\clip_vit_large_14_processor
image_encoder:
  pretrained_model: E:\ComfyUI_flux_redux\ComfyUI\models\hyper_lora/clip_vit\clip_vit_large_14
resampler:
  dim: 1024
  dim_head: 64
  heads: 12
  depth: 4
  ff_mult: 4
encoder_types: ['clip', 'arcface']
face_analyzer: antelopev2
insightface_root: E:\ComfyUI_flux_redux\ComfyUI\models\insightface
id_embed_dim: 512
num_id_tokens: 16
hyper_dim: 128
lora_rank: 8
has_base_lora: False
!!! Exception during processing !!! Error(s) in loading state_dict for CLIPVisionModelWithProjection:
        size mismatch for vision_model.embeddings.class_embedding: copying a param with shape torch.Size([1024]) from checkpoint, the shape in current model is torch.Size([768]).
        size mismatch for vision_model.embeddings.patch_embedding.weight: copying a param with shape torch.Size([1024, 3, 14, 14]) from checkpoint, the shape in current model is torch.Size([768, 3, 32, 32]).
        size mismatch for vision_model.embeddings.position_embedding.weight: copying a param with shape torch.Size([257, 1024]) from checkpoint, the shape in current model is torch.Size([50, 768]).
        size mismatch for vision_model.pre_layrnorm.weight: copying a param with shape torch.Size([1024]) from checkpoint, the shape in current model is torch.Size([768]).
        size mismatch for vision_model.pre_layrnorm.bias: copying a param with shape torch.Size([1024]) from checkpoint, the shape in current model is torch.Size([768]).
        size mismatch for vision_model.encoder.layers.0.self_attn.k_proj.weight: copying a param with shape torch.Size([1024, 1024]) from checkpoint, the shape in current model is torch.Size([768, 768]).
        size mismatch for vision_model.encoder.layers.0.self_attn.k_proj.bias: copying a param with shape torch.Size([1024]) from checkpoint, the shape in current model is torch.Size([768]).
        size mismatch for vision_model.encoder.layers.0.self_attn.v_proj.weight: copying a param with shape torch.Size([1024, 1024]) from checkpoint, the shape in current model is torch.Size([768, 768]).
        size mismatch for vision_model.encoder.layers.0.self_attn.v_proj.bias: copying a param with shape torch.Size([1024]) from checkpoint, the shape in current model is torch.Size([768]).
        size mismatch for vision_model.encoder.layers.0.self_attn.q_proj.weight: copying a param with shape torch.Size([1024, 1024]) from checkpoint, the shape in current model is torch.Size([768, 768]).
        size mismatch for vision_model.encoder.layers.0.self_attn.q_proj.bias: copying a param with shape torch.Size([1024]) from checkpoint, the shape in current model is torch.Size([768]).
........
........
.........
 size mismatch for vision_model.post_layernorm.bias: copying a param with shape torch.Size([1024]) from checkpoint, the shape in current model is torch.Size([768]).
        size mismatch for visual_projection.weight: copying a param with shape torch.Size([768, 1024]) from checkpoint, the shape in current model is torch.Size([512, 768]).
        You may consider adding `ignore_mismatched_sizes=True` in the model `from_pretrained` method.

**Comment by LemonSky1995** [comment]
确认一下CLIP的配置文件和模型是否都在`ComfyUI\models\hyper_lora/clip_vit\clip_vit_large_14`里面，以及提供一下`transformers`库的版本吧

**Comment by wuzhonghui56** [comment]
> 确认一下CLIP的配置文件和模型是否都在`ComfyUI\models\hyper_lora/clip_vit\clip_vit_large_14`里面，以及提供一下`transformers`库的版本吧

谢谢，漏了个配置文件
